src/apprt/gtk/Surface.zig

```diff
@@ -2502,3 +2502,35 @@ fn gtkStreamError(media_file: *gtk.MediaFile, _: *gobject.ParamSpec, _: ?*anyopa
 fn gtkStreamEnded(media_file: *gtk.MediaFile, _: *gobject.ParamSpec, _: ?*anyopaque) callconv(.c) void {
     media_file.unref();
 }
+
+pub fn showChildExited(self: *Surface, info: apprt.surface.Message.ChildExited) (error{})!bool {
```

Comment:
Honestly `(error{})!bool` is really funny looking. Couldn't it just be `error{}!bool`?

Reply:
Not sure, but I've had problems trying to put an explicit error set into a function signature before.

Reply:
Yeah that works just fine, not sure why I ran into problems before.